core/Rename.cpp

```diff
@@ -10,7 +10,7 @@
 #include "sparta/events/StartupEvent.hpp"
 #include "sparta/app/FeatureConfiguration.hpp"
 #include "sparta/report/DatabaseInterface.hpp"
-
+#include "mavis/ExtractorDirectInfo.h"
```

Comment:
Why do you need to include this header?

Reply:
Removed, addressed in [fbe7264](https://github.com/riscv-software-src/riscv-perf-model/pull/40/commits/fbe7264a97b4aadea92369bdf93a544c3077551e)